application/src/main/java/org/thingsboard/server/config/ThingsboardSecurityConfiguration.java

```diff
@@ -218,7 +218,7 @@ SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
                         .requestMatchers(MAIL_OAUTH2_PROCESSING_ENTRY_POINT).permitAll() // Mail oauth2 code processing url
                         .requestMatchers(DEVICE_CONNECTIVITY_CERTIFICATE_DOWNLOAD_ENTRY_POINT).permitAll() // Device connectivity certificate (public)
                         .requestMatchers(NON_TOKEN_BASED_AUTH_ENTRY_POINTS).permitAll()) // static resources, user activation and password reset end-points
-                .authorizeRequests(config -> config
+                .authorizeHttpRequests(config -> config
```

Comment:
I think one .authorizeHttpRequests is enough, plz simplify.

Reply:
We have two `authorize` each time across the code base. I am trying to understand why this is done in this way